emulation/models/orwcom-ptobj.js

```
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const orderMetaService = require('../../services/orderMetaService');
+const modelUtils = require('./modelUtils');
+const _ = require('lodash');
+
+const RPC_EMULATOR_MODEL = [
+    {
+        id: 'ORWCOM_PTOBJ',
+        name: 'ORWCOM PTOBJ',
+        function: 'PTOBJ^ORWCOM',
+        domain: 'ORDER',
+        description: 'Returns Patient COM Object',
+        rpcType: 'READ',
+        complexity: 'LOW',
+        returnType: 'ARRAY',
+        files: ['101.15'],
+        parameters: ['ORWCOM PATIENT SELECTED'],
+        labels: ['META'],
+        services: ['orderMetaService'],
+        inputParameters: [],
+        invokeRPC: (args, session, callback) => {
+            orderMetaService.lookupPatientCOMObject(session, callback);
+        },
+
+        formatResult: (res) => {
+            if (!res || _.isEmpty(res)) return new Error('expected Object param in formatResult');
+            if (res.error) return 0;
```

Hmm, I'm wondering if we should return a wrapped error instead of plain ol' `0` if there's an error?

---

Is there a way to do this without breaking the comparison-tests? The native result is 0.

---

Oh! If that's the case then yeah, this should be good! No worries...